apps/apps.humanatlas.io/src/app/pages/landing-page/landing-page.component.scss

```diff
@@ -43,19 +43,23 @@
 
   @include mat.button-toggle-overrides(
     (
-      selected-state-background-color: rgb(from var(--mat-sys-tertiary) r g b / 20%),
       text-color: var(--mat-sys-on-background),
       selected-state-text-color: var(--mat-sys-on-background),
+      background-color: var(--mat-sys-on-primary),
     )
   );
 
-  .apps-container {
-    background: linear-gradient(12.03deg, #eff1f4 -1.72%, var(--mat-sys-secondary-container) 84.47%);
+  .section-container {
     display: flex;
     flex-direction: column;
     align-items: center;
     padding: 0rem 2.5rem 6rem 2.5rem;
+    background: linear-gradient(12.03deg, #eff1f4 -1.72%, var(--mat-sys-secondary-container) 84.47%);
```

Comment:
The background is alternating between sections. Use [:nth-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) selectors to apply the correct backgrounds